### ISSUE TITLE
Geospatial mapping

### ISSUE THREAD

**Issue by ahdinosaur** [valueflows/valueflows#69]
value flows through space, here's a thread for dedicated conversion regarding this. feel free to share ramblings, share prior work, discuss ideas, propose actionable steps, or anywhere in between.

some opening questions from @iainkirkpatrick:

- how do we define space? e.g. physical vs emotional spaces
- how do we visualize space to tell the story of the data?

some previous comments relating to this topic:

- https://github.com/valueflows/exchange/issues/8
- https://github.com/valueflows/valueflows/issues/21#issuecomment-133908515

/cc @almereyda

**Comment by bhaugen** [comment]
I want to see value flows moving in physical space in realtime.

**Comment by elf-pavlik** [comment]
Myself I want to focus on physical and online spaces, very unlikely will get into modeling emotional spaces until we get some progress on physical and online...

BTW yesterday @oceatoon gave us nice demo of his new UI in https://github.com/pixelhumain with map in a background which reacts to data loaded in UI...

**Comment by bhaugen** [comment]
Link to demo or even screenshot?

**Comment by elf-pavlik** [comment]
> Myself I want to focus on physical and online spaces, very unlikely will get into modeling emotional spaces until we get some progress on physical and online...

BTW since I consider all the **valuing** aspects a subjective, mental, emotional one. I try to focus on 'just logging' what happens in physical reality (avoiding personal evaluation aspect). Later everyone can find more common or more individual ways of evaluating those happenings and introducing aspect of personal and collective **valuing**.

Sorry for going off topic but since we use term _value_ in name of this vocab and mention emotions here I thought about sharing above. In case someone see interest in further discussing **valuing** (rather than 'having value') we can create a dedicated issue for that.

**Comment by bhaugen** [comment]
Valuing and value are complicated and many-dimensional. In this value flows context, I mean economic value, by which, I mean, primarily use value.  And I mean economic in the original Greek sense of "rules of the household for good management".

As we are in a transitional stage of economic systems, it also means exchange value, which usually means monetary value, but I would like to transition out from that system as fast as possible. It is not, however, possible to avoid it now.

I don't think other realm of values or valuing apply directly here, because I don't think they are quantifiable. Which does not mean they are unimportant.

And I would be interested in how other people think about this.

**Comment by elf-pavlik** [comment]
I propose moving it to #71 I try to emphasize there that **valuing** requires an agent. With no agents we can only make statements about compatibility, dependency, applicability etc. (once again, not to hijack this issue, please let's continue that in #71)

**Comment by oceatoon** [comment]
Hi Bob , I'll try to prepare a screen cast very soon 
we are still on the polishing layer and plugging last un cabled features

**Comment by bhaugen** [comment]
@oceatoon thanks. Looking forward to it.

**Comment by almereyda** [comment]
As this is where Bob said

> I want to see value flows moving in physical space in realtime.

and I just mentioned the [realtime to offline dspace mapping environment](https://github.com/valueflows/valueflows/issues/82#issuecomment-154766819) again, imagine the following link would be embedded as in Discourse:

> https://twitter.com/Commonify/status/662945250529406976

Why there is #84 now.

**Comment by elf-pavlik** [comment]
relevant OSM tag https://wiki.openstreetmap.org/wiki/Key:ownership

**Comment by almereyda** [comment]
I am keen to think this together with http://goals.pods.wiki.org/view/welcome-visitors/view/goals-for-federated-wiki/view/top-maintenance-priorities/view/maps-and-places

While @elf-pavlik [once described](https://github.com/codeforamerica/cfapi/issues/118#issue-38716797) a nice vision of mixing Hydra and JSON-LD APIs via JSON-LD Macros.

Wonder how this can turn into a flexible engine also offering SPARQL, GeoSPARQL, bbox, turtle, LDP or even IPFS or dat, CouchDB or LevelDB replication, real time endpoints (the two latter for [gun](http://gun.js.org/)?), calendars or _what have you_ (like [SMS](https://github.com/risis-eu/sms-platform/blob/master/package.json), [API docs](http://api.risis.ops.few.vu.nl/api-config), [Geo views](https://github.com/risis-eu/LDA-apps) and later [temporal](https://github.com/kgeographer/topotime/wiki) [mapping](http://test.strabon.di.uoa.gr/sexTant/?map=mulrcpb74onu1smi_)). Wiki will lead the way to be a compatible, [fractal interface](http://blog.fract.al/) via [Linked Data Reactors](http://ld-r.org/) ([Paper](http://research.ld-r.org/adaptive-linked-data-driven-web-components.html)).

**Comment by bhaugen** [comment]
@almereyda @elf-pavlik if you can get something going on this set of ideas, I'll plug in with https://github.com/pchampin/hydra-py

**Comment by pmackay** [comment]
I'm finding this thread hard to parse - can @bhaugen or @elf-pavlik express the goals as user needs?

**Comment by bhaugen** [comment]
@pmackay my interpretation of this thread is that, as @ahdinosaur wrote in the OP above, 

> value flows through space

Or as @almereyda often reminds us, spacetime.

So I think this thread is about visualizing value flows in space and time. Space and time separately are a lot easier than together, but maybe a sequence of maps with a time slider?

I'm not sure this is a user need, but as a user, I want it.

These guys apparently focused on time first, and maps later. I'll be interested when they add the maps.
http://dh.stanford.edu/topotime/demo_py.html

**Comment by almereyda** [comment]
@kgeographer is currently revising topotime, also see http://kgeographer.com/hello-world/#comments
